stl/src/vector_algorithms.cpp

```diff
@@ -1114,12 +1112,7 @@ namespace {
         }
 
         static uint64_t _Get_any_u(const __m128i _Cur) noexcept {
-#ifdef _M_IX86
-            return (static_cast<uint64_t>(static_cast<uint32_t>(_mm_extract_epi32(_Cur, 1))) << 32)
-                 | static_cast<uint64_t>(static_cast<uint32_t>(_mm_cvtsi128_si32(_Cur)));
-#else // ^^^ x86 / x64 vvv
-            return static_cast<uint64_t>(_mm_cvtsi128_si64(_Cur));
-#endif // ^^^ x64 ^^^
+            return _Minmax_traits_8::_Get_any(_Cur);
         }
 
         static uint64_t _Get_v_pos(const __m128i _Idx, const unsigned long _H_pos) noexcept {

```

Comment:
Pre-existing: `_Minmax_traits_8` and `_Minmax_traits_d` have exactly identical `_Get_v_pos` (modulo typedef usage), which makes sense since `_Get_any_u` and `_Get_v_pos` are the parts that still behave like integers (whereas the other member functions were `double`-ified). We should just call `_Minmax_traits_8` here.